core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -691,6 +691,18 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  private[spark] val KERBEROS_FILESYSTEM_RENEWAL_EXCLUDE =
+    ConfigBuilder("spark.kerberos.renewal.exclude.hadoopFileSystems")
```

Comment:
I also think we should change the name fo this.  we have spark.kerberos.renewal.credentials that is talking about how Spark renewals. This has the same prefix but is not about spark renewal, its about resource scheduler renewal.  
Honestly at this point if no one knows of a k8s component that could do this I would almost say we make the config yarn specific - spark.yarn.kerberos.renewal.excludeHadoopFileSystems. I would actually like to see rm or resourceScheduler in the name to make it even more clear but that gets really long.      Open to other opinions but we can always make a generic one later. If someone has an idea how this would be used with others we can leave it generic to not yarn but I want something else in the name to show its resource scheduler specific.

Reply:
`spark.yarn.kerberos.renewal.excludeHadoopFileSystems` looks okay to me.